lib/graph3d/Graph3d.js

```diff
@@ -2365,12 +2363,12 @@ function getMouseY (event) {
  * Set the rotation and distance of the camera
  *
  * @param {Object}  pos            An object with the camera position
- * @param {?Number} pos.horizontal The horizontal rotation, between 0 and 2*PI.
+ * @param {number} [pos.horizontal] The horizontal rotation, between 0 and 2*PI.
  *                                 Optional, can be left undefined.
- * @param {?Number} pos.vertical   The vertical rotation, between 0 and 0.5*PI.
+ * @param {number} [pos.vertical]  The vertical rotation, between 0 and 0.5*PI.
  *                                 if vertical=0.5*PI, the graph is shown from
  *                                 the top. Optional, can be left undefined.
- * @param {?Number} pos.distance   The (normalized) distance of the camera to the
+ * @param {number} [pos.distance]  The (normalized) distance of the camera to the
```

Comment:
[]-notation is used for default values. Look up before-mentioned reference for proper syntax.
Or I might be missing some point here, again.

Reply:
Again, using old jsdoc reference bit me. The given change is legal.